File: Heat/Models/BoilerHeater.vb
Namespace Models
    ''' <summary>
    ''' Rappresenta un bruciatore della caldaia; può essere l'unico bruciatore o uno dei tanti bruciatori.
    ''' </summary>
    ''' <remarks></remarks>
    Public Class BoilerHeater
        Property ID As Integer
        Property Boiler As Boiler
        Property Manifacturer As Manifacturer
        Property Model As ManifacturerModel
        Property SerialNumber As String
        Property MinimumPowerKW As Single
        Property MaximumPowerKW As Single
        Property CertificationReference As String

    End Class
End Namespace

File: Heat.ConvertedToC#/Models/BoilerHeater.cs
namespace Heat.Models
{
    /// <summary>
    /// Rappresenta un bruciatore della caldaia; può essere l'unico bruciatore o uno dei tanti bruciatori.
    /// </summary>
    /// <remarks></remarks>
    public class BoilerHeater
	{
		public int ID { get; set; }
		public Boiler Boiler { get; set; }
		public Manifacturer Manifacturer { get; set; }
		public ManifacturerModel Model { get; set; }
		public string SerialNumber { get; set; }
		public float MinimumPowerKW { get; set; }
		public float MaximumPowerKW { get; set; }
		public string CertificationReference { get; set; }

	}
}
